### PR TITLE
Qute - relax infix notation syntax

### PR DESCRIPTION
**Pull request by mkouba**
- ignore redundant blank space before infix method and before infix
method param

### CODE REVIEW

**Comment by mkouba** [comment]
CC @angelozerr

**Comment by mkouba** [comment]
@gastaldi Pls review ;-)

File: independent-projects/qute/core/src/main/java/io/quarkus/qute/Expressions.java [hunk 1]
```diff
@@ -100,10 +100,20 @@ public static String buildVirtualMethodSignature(String name, List<String> param
...
-                        if (infix == 1) {
-                            // The second space after the infix method
+                        // Infix supported, blank space and not inside a virtual method
+                        if (separator == 0
+                                && (buffer.length() == 0 || buffer.charAt(buffer.length() - 1) == '(')) {
+                            // Skip redundant blank space:
+                            // 1. before the infix method
+                            // foo  or bar
+                            // ----^
+                            // 2. before an inifix method parameter
```
Comment (gastaldi) [inline independent-projects/qute/core/src/main/java/io/quarkus/qute/Expressions.java:112]:
```suggestion
                            // 2. before an infix method parameter
```

**Review by gastaldi — APPROVED** [review]
LGTM

**Pull request merged**